common/assets/sass/application.scss

```diff
@@ -21,8 +27,12 @@ $white-transparent: rgb(255 255 255 / 25%);
   padding-top: 30px;
 }
 
-.govuk-width-container--app-extra-wide {
-  max-width: 1220px;
+.use-govuk-page-width {
```

Comment:
Created this style because we're using precompiled CSS for the MoJ Design System so we don't pick up `govuk-page-width` for their styles